docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,48 @@
 
-Copy the config.js file from the `app` folder in the latest kit to the `app` folder of your prototype. If you've made any changes to the config.js file in your prototype then you'll need to re-enter them in the new version of the file e.g. the service name.
+8. Copy `app/assets/sass/patterns` from the unzipped folder to your prototype.
 
-Check `\app\assets\sass\patterns` in the latest kit for any new patterns. Copy the files over to your prototype.
+9. Copy `app/assets/sass/components` from the unzipped folder to your prototype.
 
-Check `\app\assets\sass\application.scss` in the latest kit to see if any changes have been made in the top section, above where it says `// Add extra styles here`. Copy anything new from that file to the version in your prototype, making sure you don't overwrite any extra styles you have added yourself.
+10. Open the `app/assets/sass/application.scss` file in the unzipped folder.
 
-Check the [release notes for the latest release](https://github.com/alphagov/govuk-prototype-kit/releases/latest), for any additional changes you need to make to your prototype.
+11. Copy everything down to `// Add extra styles here`, then paste it into the `app/assets/sass/application.scss` file in your prototype so it replaces everything above `// Add extra styles here`.
 
----
+12. Save the updated `app/assets/sass/application.scss` file in your prototype.
 
-## Updating via the command line (Advanced)
+13. Check the [latest Prototype Kit release note](https://github.com/alphagov/govuk-prototype-kit/releases/latest) and follow any guidance on updating your prototype.
 
-If you have experience with the command line and your prototype is running within a git source repository then you can choose to update the kit with a series of commands.
+    If your prototype has not been updated for a long time, you should also follow any guidance in [release notes](https://github.com/alphagov/govuk-prototype-kit/releases) between the version you're updating from and the latest version. You can find out the version you're updating from in the `VERSION.txt` file in your backup folder.
 
-Updating via the command line involves fetching the latest code from the 'upstream' remote repository and then merging it into the git branch that contains your version of the prototyping kit. Below are the steps for doing that in detail.
+14. In your [terminal](https://govuk-prototype-kit.herokuapp.com/docs/install/requirements.md#terminal), `cd` to your prototype folder.
 
-### Steps
+15. Run `npm install`.
 
-#### View git remote(s)
+    This may take up to a minute. You can ignore any lines in the log that start with `WARN`.
 
-Firstly change to the base directory of your prototyping kit in terminal, for example:
+16. [Run the kit and check it works](/docs/install/run-the-kit).
 
-```
-cd ~/sites/govuk-prototype-kit
-```
+## Get help
```

Comment:
@m-green Noticed that this heading is not rendering correctly, probably because the hash characters were copied over from a doc or similar. 

<img width="347" alt="Screenshot 2020-07-08 at 09 44 10" src="https://user-images.githubusercontent.com/5007934/86898153-3fa28e00-c100-11ea-92a1-6fa7457e1890.png">

The hashes just need to be replaced with a code editor to fix this 🙂

Reply:
Hi @hannalaakso  - that heading seems to work ok for me? 🤔 

![image](https://user-images.githubusercontent.com/20596506/86900763-a70e0d00-c103-11ea-90e9-108f125964e4.png)

Reply:
@hannalaakso Don't worry - Joe fixed it! Looks like it was the space after the hashes rather than the hashes themselves.